terraform_storage_account/create-remote-stroage.tf

```diff
@@ -0,0 +1,37 @@
+terraform {
```

Comment:
We should only track a single method for creating the storage account - we wouldn't usually track the remote storage with Terraform since it introduces confusion (e.g. where does the state for this exist?) so I'd advise removing this and sticking with the bash file that you've written and referenced in the docs